src/tools/slack-alert-config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "active": true,
```

Comment:
false by default. This is because we do not set default token so the deployer should change it to true when setting the token